nexus/src/app/quiesce.rs

```diff
@@ -92,6 +92,12 @@ impl NexusQuiesceHandle {
         self.state.borrow().clone()
     }
 
+    pub fn latest_blueprint(
+        &self,
+    ) -> Option<Arc<(BlueprintTarget, Blueprint)>> {
+        self.latest_blueprint.borrow().clone()
+    }
```

Comment:
It seems strange to expose this from here.  If somebody else needs this, they should probably just have their own `watch::Receiver` for this.